base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbt.java

```diff
@@ -203,45 +204,53 @@ public Image getNextTurnIcon() {
      *
-     * @param distanceToManeuver a Double value representing a Fraction of distance till next maneuver
-     *                           <p></p>
-     *                           <b>Notes: </b>Minvalue=0; Maxvalue=1000000000
+     * @param distanceToManeuver Distance (in meters) until next maneuver. May be used to calculate progress bar.
+     * {"num_min_value": 0.0, "num_max_value": 1000000000.0}
+     * @since SmartDeviceLink 2.0.0
      */
     public ShowConstantTbt setDistanceToManeuver(Double distanceToManeuver) {
-        setParameters(KEY_MANEUVER_DISTANCE, distanceToManeuver);
+        setParameters(KEY_DISTANCE_TO_MANEUVER, distanceToManeuver);
         return this;
     }
 
     /**
-     * Gets a Fraction of distance till next maneuver
+     * Gets the distanceToManeuver.
      *
-     * @return Double -a Double value representing a Fraction of distance till next maneuver
+     * @return Float Distance (in meters) until next maneuver. May be used to calculate progress bar.
+     * {"num_min_value": 0.0, "num_max_value": 1000000000.0}
+     * @since SmartDeviceLink 2.0.0
      */
     public Double getDistanceToManeuver() {
-        return getDouble(KEY_MANEUVER_DISTANCE);
+        Object object = getParameters(KEY_DISTANCE_TO_MANEUVER);
+        return SdlDataTypeConverter.objectToDouble(object);
     }
 
     /**
-     * Sets a Distance till next maneuver (starting from) from previous maneuver
+     * Sets the distanceToManeuverScale.
      *
-     * @param distanceToManeuverScale a Double value representing a Distance till next maneuver (starting from) from previous maneuver
-     *                                <p></p>
-     *                                <b>Notes: </b>Minvalue=0; Maxvalue=1000000000
+     * @param distanceToManeuverScale Distance (in meters) from previous maneuver to next maneuver. May be used to calculate
+     * progress bar.
+     * {"num_min_value": 0.0, "num_max_value": 1000000000.0}
+     * @since SmartDeviceLink 2.0.0
      */
     public ShowConstantTbt setDistanceToManeuverScale(Double distanceToManeuverScale) {
-        setParameters(KEY_MANEUVER_DISTANCE_SCALE, distanceToManeuverScale);
+        setParameters(KEY_DISTANCE_TO_MANEUVER_SCALE, distanceToManeuverScale);
         return this;
     }
 
     /**
-     * Gets a Distance till next maneuver (starting from) from previous maneuver
+     * Gets the distanceToManeuverScale.
      *
-     * @return Double -a Double value representing a Distance till next maneuver (starting from) from previous maneuver
+     * @return Float Distance (in meters) from previous maneuver to next maneuver. May be used to calculate
+     * progress bar.
+     * {"num_min_value": 0.0, "num_max_value": 1000000000.0}
+     * @since SmartDeviceLink 2.0.0
      */
     public Double getDistanceToManeuverScale() {
-        return getDouble(KEY_MANEUVER_DISTANCE_SCALE);
+        Object object = getParameters(KEY_DISTANCE_TO_MANEUVER_SCALE);
```

Comment:
@kostyaBoss  I would delete lines 252-253 and replace it with the old code `return getDouble(KEY_MANEUVER_DISTANCE_SCALE);`.